woocommerce/woocommerce-ios
# [Better Customer Selection] Enable Feature

Closes: #10313 


## Description

With this PR we enable the Better Customer Selection feature so it can be shown in Release.


## Testing instructions


Run the app in Release build configuration and ensure the feature is visible. You can set the Release build configuration by editing the WooCommerce schema (see screenshot).

## Screenshots


<img width="928" alt="Screenshot 2023-07-17 at 10 44 14" src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/676ff06d-352c-431d-b8e5-9545eaee0753">


---
